### PR TITLE
Remove unused files, unused imports, commented out code, and update NPM packages

### PR DESCRIPTION
**Pull request by JoshVarga**
General cleanup

### CODE REVIEW

File: src/simulator/src/modules/Ground.js [hunk 1]
```diff
@@ -54,21 +50,6 @@ export default class Ground extends CircuitElement {
         simulationArea.simulationQueue.add(this.output1)
     }
 
-    /**
-     * @memberof Ground
-     * fn to create save Json Data of object
-     * @return {JSON}
-     */
-    customSave() {
```
Comment (gr455) [inline src/simulator/src/modules/Ground.js:62]: I believe this definition of customSave is being used currently. The earlier one is redundant (the Ground constructor does not take the direction, and Ground does not have a state).

File: src/simulator/src/modules/Output.js [hunk 2]
```diff
@@ -59,15 +56,6 @@ export default class Output extends CircuitElement {
         this.inp1 = new Node(this.bitWidth * 10, 0, 0, this)
     }
 
-    /**
-     * @memberof Output
-     * function to generate verilog for output
-     * @return {string}
-     */
-    generateVerilog() {
```
Comment (gr455) [inline src/simulator/src/modules/Output.js:67]: This function generates verilog for the output node, removing it will break that feature.

File: src/simulator/src/modules/Splitter.js [hunk 3]
```diff
@@ -183,53 +178,12 @@ export default class Splitter extends CircuitElement {
         this.prevInpValue = undefined
     }
 
-    /**
-     * @memberof Splitter
-     * fn to process verilog of the element
-     * @return {JSON}
-     */
-    processVerilog() {
```
Comment (gr455) [inline src/simulator/src/modules/Splitter.js:191]: Why was this removed?

File: src/simulator/src/modules/Ground.js (hunk 1 above)
Comment (JoshVarga) [reply src/simulator/src/modules/Ground.js:62]: Currently Ground.js has customSave() defined twice, in javascript the second one wins

Comment (gr455) [reply src/simulator/src/modules/Ground.js:62]: Yep, we would want the second one to keep winning since that is the correct implementation. So I think it would be better to remove the [first implementation](https://github.com/CircuitVerse/cv-frontend-vue/blob/d0787734b8183f83f67d11c31c6f888c7c8f8058/src/simulator/src/modules/Ground.js#L31)

File: src/simulator/src/modules/Splitter.js (hunk 3 above)
Comment (JoshVarga) [reply src/simulator/src/modules/Splitter.js:191]: Here too generateVerilog is defined twice for the same object

File: src/simulator/src/modules/Output.js (hunk 2 above)
Comment (gr455) [reply src/simulator/src/modules/Output.js:67]: Gotcha. This is good then. Thanks

File: src/simulator/src/modules/Splitter.js (hunk 3 above)
Comment (gr455) [reply src/simulator/src/modules/Splitter.js:191]: Got it.

**Review by gr455 — APPROVED** [review]
LGTM

**Comment by gr455** [comment]
@tachyons Can you merge this. I don't seem to have write access. (also please give me write access too)
